saleor/channel/models.py

```diff
@@ -6,6 +6,7 @@
 
 class Channel(models.Model):
     name = models.CharField(max_length=250)
+    is_active = models.BooleanField(default=True)
```

Comment:
I'm not sure that we should make the channel as active by default.

Reply:
Agree, we can make it inactive, just as we do with newly created products. Once users decide they want to publish the channel they'll do it explicitly.